system/jlib/jmetrics.cpp

```diff
@@ -153,6 +153,13 @@ bool MetricsManager::addMetric(const std::shared_ptr<IMetric> &pMetric)
     bool rc = false;
     std::string name = pMetric->queryName();
 
+    // Remove unwanted characters from input name
+    char removeChars[] = "_-* ";
+    for (unsigned i=0; i<strlen(removeChars); ++i)
+    {
+        name.erase(std::remove(name.begin(), name.end(), removeChars[i]), name.end());
```

Comment:
@kenrowland Should some or all of these characters be replaced with '.' instead of removing them?  Is more than on '.' in a row supported.

Reply:
No, periods are the official name space separator used by the framework. The Sinks use the period to parse names in order to build their reports.